config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
+  devise_for :users, controllers: { registrations: 'registrations' }
   resources :posts
```

Comment:
It was asked to set it for 3 routes only

![Screenshot_2020-04-22_21-23-11](https://user-images.githubusercontent.com/8848104/80029996-85778f00-84df-11ea-859a-222c8b8d57c0.png)

Reply:
added only %i[new create index]